src/xpath.c

```diff
@@ -160,22 +162,44 @@ static void
 print_expr_struct_debug(struct lyxp_expr *exp)
 {
     uint16_t i, j;
-    char tmp[128];
+    char tmp[DBG_BUFF_SIZE];
+    char *buff = tmp;
+    int buff_left = DBG_BUFF_SIZE - 1;
```

Comment:
`buff_left` is redundant, it is always `DBG_BUFF_SIZE - 1 - buff_used`, right? But okay, I suppose, not a bug deal.